src/main/java/us/icarenow/cnsas/model/AssuranceStatus.java

```diff
@@ -0,0 +1,28 @@
+package us.icarenow.cnsas.model;
+
+import java.util.Arrays;
+
+public enum AssuranceStatus {
+    ASSURED("1890511125802", "assured"),
+    NOT_ASSURED("40364512464", "not_assured"),
+    NOT_FOUND("404576513265", "not_found");
+
+    private String cnp;
```

Comment:
let's change this a little bit to make more dynamically.
this could be cnpPrefix instead of cnp, and we could consider all cnps starting with 187 to be assured,  with 295 to be not_assured and in rest by default they will be considered not found